### PR TITLE
Added section about the review process

### PR DESCRIPTION
**Pull request by JamieRpg**
### Description

Added a section about the review steps, when creating a pull-request in Github.

### Type of Change

- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/git/concepts/pull-requests/pull-requests.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Pull requests (PRs) are used to recommend the merging of code from one [branch](
 
 After reviewing, if the changes are approved by the author(s)/collaborators of the repository, they are merged into the base branch (this is usually named the `main` branch). If the changes are not approved, the maintainer(s) may either ask the requester for additional changes or decline the PR altogether.
 
+The reviewing process by the author(s)/collaborators, usually involves these steps:
```
Comment (yangc95) [inline content/git/concepts/pull-requests/pull-requests.md:19]:
```suggestion
The review process by the contributors, usually involves these steps:
```

File: content/git/concepts/pull-requests/pull-requests.md [hunk 2]
```diff
@@ -16,6 +16,12 @@ Pull requests (PRs) are used to recommend the merging of code from one [branch](
...
+
+1. Examining the code submitted in the PR for errors and/or improvements.
```
Comment (yangc95) [inline content/git/concepts/pull-requests/pull-requests.md:21]:
```suggestion
1. Examining the code submitted in the PR for errors and/or optimizations.
```

Comment (yangc95) [reply content/git/concepts/pull-requests/pull-requests.md:21]: can we make this a bulleted list?

**Review by yangc95 — APPROVED** [review]
@JamieRpg approving this for a second review. I'd like to also leave a note here for the next reviewer that I think this is generalized enough where it's valid information, although it's important to note that each repo's review process differs.

**Review by SSwiniarski — APPROVED** [review]
@JamieRpg looks good.

**Comment by SSwiniarski** [comment]
@JamieRpg Congrats on the new edit.

Live link:
- https://www.codecademy.com/resources/docs/git/pull-requests

**Pull request merged**